ResetPasswordBundle/src/Resources/config/password_reset_services.xml

```diff
@@ -5,12 +5,16 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <services>
+        <service id="symfonycasts.reset_password.token_generator" class="SymfonyCasts\Bundle\ResetPassword\Generator\TokenGenerator" public="false" />
+
+        <service id="SymfonyCasts\Bundle\ResetPassword\PasswordResetHelperInterface" alias="symfonycasts.reset_password.helper" />
+
         <service id="symfonycasts.reset_password.helper" class="SymfonyCasts\Bundle\ResetPassword\PasswordResetHelper">
             <argument /> <!-- password reset request persister -->
-            <argument>%kernel.secret%</argument>
+            <argument>%kernel.secret%</argument> <!-- signing key -->
             <argument /> <!-- reset request lifetime -->
+            <argument /> <!-- reset throttle limit -->
+            <argument type="service" id="symfonycasts.reset_password.token_generator"/> <!-- token generator -->
```

Comment:
We don't usually include the `<!--` comments unless a value looks empty here, but will be filled in later dynamically (that's the case for the first argument).

Reply:
done